README.md

```diff
@@ -133,6 +133,23 @@ typically in the the `<head>`. The syntax for using a `<link>` tag is `<link
 rel="stylesheet" type="text/css" href="...">` where 'href' is set to the
 location of the desired stylesheet.
 
+### Code-Along: Barry's Butcher Shop
+
+Together we are going to look at our good friend Barry's purposal for his
```

Comment:
this one too @berziiii 
We can talk about if there's a better way to do this...